Docker/OutputFilter/OutputFilter.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Keboola\DockerBundle\Docker\OutputFilter;
+
+class OutputFilter implements OutputFilterInterface
+{
+    const REPLACEMENT = '*****';
+
+    /**
+     * @var array
+     */
+    private $filterValues = [];
+
+    /**
+     * @inheritdoc
+     */
+    public function addValue($value)
+    {
+        $this->filterValues[] = $value;
```

Comment:
jen myšlenka - co když bude potřeba tu hodnotu ještě nějak víc modifikovat než jen `base64`? hodnoty a modifiery bych od sebe oddělil, takže třeba

```php
$filter->addValue('mypassword');
$filter->addModifier(function($value) {
    return base64_encode($value);
});
```

```php
    public function filter($text)
    {
        foreach ($this->filterValues as $filterValue) {
            foreach ($this->filterModifiers as $filterModifier) {
                $text = str_replace($filterModifier($filterValue), self::REPLACEMENT, $text);
            }
        }
        return $text;
    }
```

Reply:
je v tom trosku premature optimization - toho textu, kterej se muze filtrovat muze byt pomerne hodne, takze jsem sel radsi cestou predpocitanejch replacementu nez delat to on-fly

jinak je otazka jestli tam nejaky dalsi 'modifikatory' vubec muzou byt, napadla me tak maximalne md5, ktera je tak insecure, ze je skoro plaintext :) ale vubec si nedovedu predstavit nejakje usecase

Reply:
jsem si unerndnul, souhlas. odkud se tam ty base64 dostávaj?

Reply:
nevim, jedine u genericu a prijde mi, ze to asi tak jedina moznost jak predat neco plaintext a pritom to nebyt plaintext, muzem to klidne vyhodit jestli myslis, ze je to zbytecny

Reply:
můžem projet stávající configy a zkusit z nich zjistit, co se tam používá?

Reply:
to z konfigu nijak nezjistis, filtruje to prece eventy

Reply:
ale může se zjistit, co to používá za funkce a jak zkombinovaný :-)

Reply:
dal bych tam cokoliv, o čem teď víme, že se nějak víc používá.